linuxdeploy-plugin-gtk.sh

```diff
@@ -305,3 +308,19 @@ for directory in "${PATCH_ARRAY[@]}"; do
         ln $verbose -s "${file/\/usr\/lib\//}" "$APPDIR/usr/lib"
     done < <(find "$directory" -name '*.so' -print0)
 done
+
+# set write permission on lib64 again to make it deletable.
+chmod +w "$APPDIR"/usr/lib64 || true
+
+# We have to copy the files first to not get permission errors when we assign gio_extras_dir
+find /usr/lib* -name libgiognutls.so -exec mkdir -p "$APPDIR"/"$(dirname '{}')" \; -exec cp --parents '{}' "$APPDIR/" \; || true
+# related files that we seemingly don't need:
+# libgiolibproxy.so - libgiognomeproxy.so - glib-pacrunner
+
+gio_extras_dir=$(find "$APPDIR"/usr/lib* -name libgiognutls.so -exec dirname '{}' \; 2>/dev/null)
```

Comment:
It is not safe, it can returns multiples directories, like:
```
/usr/lib/gio/modules
/usr/lib32/gio/modules
```

Use `"$(get_pkgconf_variable "giomoduledir" "gio-2.0")` instead.